### PR TITLE
Add warning to nextcloud_apps.py

### PR DESCRIPTION
**Pull request by alejandroscf**
I've added the 'update available' warning to the non-multigraph plugin.

### CODE REVIEW

File: nextcloud_apps.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ def config(self):
 				'num_updates_available.label available app updates',
 				'num_updates_available.info number of available app updates',
 				'num_updates_available.min 0'
```
Comment (mightyBroccoli) [inline nextcloud_apps.py]:
```suggestion
				'num_updates_available.min 0',
```

**Review by mightyBroccoli — CHANGES_REQUESTED** [review]
Mind the comma to separate dict entries.

**Comment by mightyBroccoli** [comment]
I have to say that I did this to my local installation too. But I was not really sure what to set the warning level/ amount to. 
Some instances with docents of apps might want a way higher warning limit.
I am quite unsure about the number.
A customisable number could grant some customisability through an env variable.

**Comment by alejandroscf** [comment]
The moment you have an update available, you may have a security issue until you update that app, so I think that always rising a warning is a good move.

**Comment by mightyBroccoli** [comment]
I think that your approach is better then mine. If there people start to complain with a reason the change could be done.

**Review by mightyBroccoli — APPROVED** [review]

**Pull request merged**